packages/features/auth/signup/handlers/selfHostedHandler.ts

```diff
@@ -78,7 +80,26 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
         organizationSettings: true,
       },
     });
+
     if (team) {
+      const isInviteForOrganization = isOrganization({
+        team: {
+          metadata: team.metadata,
+        },
+      });
```

Comment:
Since #13700 added the `isOrganization` property to the team record. Can we remove selecting `organizationSettings` from the team query and just select the `isOrganization` field under the parent?

Reply:
Inner functions that get called still need the settings but I've changed the logic to use `isOrganization` in a clean way.